tests/robot-tests/tests/libs/snapshots.py

```diff
@@ -20,27 +20,25 @@
 
 slack_service = SlackService()
 
+public_url = os.getenv("PUBLIC_URL")
```

Comment:
If you pull this out, you can remove all instances of `os.getenv('PUBLIC_URL')` from this file.